.github/workflows/tests.yaml

```diff
@@ -36,7 +36,7 @@ jobs:
           sudo echo "127.0.0.1 materialize" | sudo tee -a /etc/hosts
 
       - name: Start Materialize in Docker
-        uses: isbang/compose-action@v1.4.1
```

Comment:
The action seems to have been moved under a new organization causing the CI tests to fail. Just updating the reference to the new org and bumping the version to the latest one as well.